src/holofuel/components/CopyToClipboard/CopyToClipboard.js

```diff
@@ -0,0 +1,31 @@
+import { string } from 'prop-types'
+import React from 'react'
+import copy from 'copy-to-clipboard'
+import './CopyToClipboard.module.css'
+import useFlashMessageContext from 'holofuel/contexts/useFlashMessageContext'
+
+export default function CopyToClipboard ({
+  copyContent,
+  messageText,
+  children
+}) {
+  if (!messageText) messageText = `${copyContent} has been copied!`
```

Comment:
I like the default.

Neat trick, you can do this on line 9
```
messageText = `${copyContent} has been copied!`
```
will set that as the default, which will be overwritten if a value is passed in.